src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/equalshashcode/InputEqualsHashCodeSemantic.java

```diff
@@ -93,7 +93,7 @@ public int hashCode()
 
         ByteArrayOutputStream bos2 = new ByteArrayOutputStream()
         {
-            public boolean equals(Object a) // violation
+            public boolean equals(Object a) // violation 'Definition of 'equals()' without corresponding definition of 'hashCode()'.'
```

Comment:
Please keep line length to be less than 100 symbols.
Let's enforce this rule by Checkstyle.
Only very limited files should be allowed to exceed this limit

Reply:
This rule is already enforced.
https://github.com/checkstyle/checkstyle/blob/5d32b56020c5b75b8aa786c4c6de55f38573796f/config/checkstyle_non_main_files_checks.xml#L18-L24

Reply:
why it is not violated ?

`.*\/\/ violation` should be removed, it was added when we only started doing this and it was slight extension of line only. So now we overuse comments a lot. review of code become problematic with such horizontal scroll.
Please remove this part of regexp and move all existing violations to suppress file.

Reply:
> Message verification should be compressed by regexp.

done

Reply:
> .*\/\/ violation should be removed, it was added when we only started doing this and it was slight extension of line only. 

even after compression violation comments are around 40 characters long. If we enforce Checkstyle for this, it'll cause a lot of nuances. I think it's best to suppress it and at the same time, try to use regexp to shorten messages as much as possible.

Reply:
There are a lot of rooms .... to shorten it.
change indentaiton to +2, in whole file, change names, take most representative part from message.

Reply:
> change indentaiton to +2

this will mess up all the column numbers in the expected array.

> change names

same problem. the exected array will need changes. Moreover, identifiers are anyway short in inputs usually.

> take most representative part from message

done that in this PR, still around 40 characters.

Reply:
It will not mess up, it will change column numbers. We do not afraid of doing this, when necessary.